charts/aws-calico/README.md

```diff
@@ -0,0 +1,66 @@
+# Calico on AWS
+
+This chart installs Calico on AWS: https://docs.aws.amazon.com/eks/latest/userguide/calico.html
+
+## Prerequisites
+
+- Kubernetes 1.11+ running on AWS
+
+## Installing the Chart
+
+First add the EKS repository to Helm:
+
+```shell
+helm repo add eks https://aws.github.io/eks-charts
+```
+
+Install the Calico CRDs:
+
+```shell
+kubectl apply -k github.com/aws/eks-charts/tree/master/stable/aws-calico/crds
+```
+
+To install the chart with the release name `aws-calico` and default configuration:
+
+```shell
+$ helm install --name aws-calico --namespace kube-system eks/aws-calico
```

Comment:
This is the helm2 install command.  We updated this not too long ago in the aws-node-termination-handler readme.  It's probably worth it to change to the helm3 version. Keeping the helm2 instructions in addition to helm3 is up to you. 

`
#Helm2
$ helm install --name aws-calico --namespace kube-system eks/aws-calico
#Helm3 
$ helm install aws-calico --namespace kube-system eks/aws-calico
`

Reply:
Sure will update it in the following PR. There are 2 AIs for the following PR - calico multi arch and helm3 in README.